Parent/Parent/ParentAppDelegate.swift

```diff
@@ -224,6 +231,15 @@ extension ParentAppDelegate: LoginDelegate {
             userDidLogout(session: session)
         }
     }
+
+    static func persistCurrentStudent(id: String?) {
+        UserDefaults.standard.set(id, forKey: currentStudentDefaultsKey)
+        UserDefaults.standard.synchronize()
```

Comment:
FYI `synchronize()` shouldn't be called anymore. From [the docs](https://developer.apple.com/documentation/foundation/userdefaults/1414005-synchronize):

> this method is unnecessary and shouldn't be used.

Reply:
thanks, didn't know, will remove